Week-4/Assignment-2/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+class Person 
```

Comment:
in javascript use braces like this

class Person {

}

The reason is most of the developers use this way of putting the braces, even later when you will be introduced to the linters, that will also follow the same.